### ISSUE TITLE
Markdown.ToHtml throws ArgumentOutOfRangeException

### ISSUE THREAD

**Issue by Metalnem** [lunet-io/markdig#275]
**Markdown.ToHtml** throws an **ArgumentOutOfRangeException** when converting the markdown file from the [attached archive]. You can run the following code to reproduce it (the **path** variable should contain the path to the extracted file):

```csharp
var text = File.ReadAllText(path);
var pipeline = new MarkdownPipelineBuilder().UseAdvancedExtensions().Build();
Markdown.ToHtml(text, pipeline);
```

Found via [SharpFuzz].

[attached archive]: https://github.com/lunet-io/markdig/files/2707155/ArgumentOutOfRangeException.zip
[SharpFuzz]: https://github.com/Metalnem/sharpfuzz

**Comment by xoofx** [comment]
Please stop reporting with an automated tool. I don't have time to download an archive and check the issues. So I prefer to have a simple code snippet with a reproducible error. Thank you

**Comment by xoofx** [comment]
The issue should be fixed by commit 03858dc

I would appreciate if next time you can include the full exception directly in your report (to give more context). 

Thanks for the fuzz testing. my apologize if I was angry when receiving these notifications but that was really a bad timing for me getting bombarded with issues while preparing Christmas diner 😅 

The fact that the report looks like an automated stuffs was really scary for me at first, I thought I would receive hundred of them in the following minutes, hence blocking you straight.

**Comment by Metalnem** [comment]
My bad about the missing stack trace! I started including them in reports for the other repositories the next day 😃
